portfolio/src/main/webapp/index.html

```diff
@@ -8,14 +8,83 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <h2>Theodora-Mara Pislar</h2>
-      <p>This is my portfolio.</p>
-      <p>Here you will find more things about me.</p>
+      <h1 style = "text-align : center;">My Portfolio</h1>
+      <h2 style = "text-align : center;">Theodora-Mara Pîslar</h2>
+      <p>You can find me here:
+        <ul>
+         <li> <a href=
+         "https://www.linkedin.com/in/theodora-mara-pislar-983117196/"
+         target="_blank"> 
+          LinkedIn</a> </li> 
+         <li> <a href="https://github.com/maraPislar"
+         target="_blank">
+          GitHub</a> </li>
+        </ul>
+      </p>
+      <p>
+          Keep scrolling to find more!
+      </p>
       <img src="/images/me.jpeg">
       <p>Click here to get a random fact about me:</p>
       <button onclick="addRandomFact()">Fact</button>
       <div id="fact-container"></div>
+      <h3>Background</h3>
+      <h4>Residence</h4>
+      <p>
+          Since I was a small kid, I have lived in Radauti. 
+          Radauti is a small town from the North of Romania which is
+          very popular in the country because of a soup. This soup
+          takes over its name ("Ciorba Radauteana") because it 
+          was invented here and is served in most of the restaurants 
+          all around the country. <! -- Insert picture with town -->
+      </p>
+      <h4>Family</h4>
+      <p>
+          I was born on the 25th of July in 2000 in a loving family.
+          I have two other sisters, me being the middle child. Miruna
+          is 24 and Sofia is 5 at the moment. We love to spend time
+          together. <! -- Insert picture with sisters -->
+      </p>
+      <h4>Education</h4>
+      <p>
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Indentation
The standard for indentation is 2 spaces.